sdlib/d/gc/page.d

```diff
@@ -11,6 +11,8 @@ import d.gc.util;
 
 import sdc.intrinsics;
 
+extern(C) void __sd_run_finalizer(void* ptr, size_t usedSpace, void* finalizer);
```

Comment:
You should import that declaration instead of duplicating it all over the place.

Reply:
I had already declared it in d/finalizer.d, but I didn't know how the linking worked. I thought if I imported that file, it would include the implementation, not realizing the object is not built for it.